vsphere/manifest.json

```diff
@@ -15,7 +15,9 @@
   "guid": "930b1839-cc1f-4e7a-b706-0e8cf3218464",
   "public_title": "Datadog-vSphere Integration",
   "categories":["cloud"],
+  "type":"check",
   "doc_link": "https://docs.datadoghq.com/integrations/vsphere/",
   "is_public": true,
-  "has_logo": true
+  "has_logo": true,
+  "aliases":["/integrations/vmware/"]
```

Comment:
Is the the ending `/` is needed ? or is it a typo ?

Reply:
it's needed :)